project_7.html

```diff
@@ -725,19 +736,54 @@ <h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
 							<td>1</td>
 						</tr>
 						<tr>
-							<td>Micro USB Cable</td>
+							<td>
+								<a class="pj-part"
+									data-bs-content="<img src='./assets/images/project_4/ops_micro_usb.jpg' height='150px' width='auto'>"
+									data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+									Nano-compatible USB Cable
+								</a>
+							</td>
 							<td>1</td>
 						</tr>
 						<tr>
-							<td>TM1637, 7 Segment Display</td>
+							<td>
+								<a class="pj-part"
+									data-bs-content="<img src='assets\images\project_7\ops_TM1637_physical.png' height='150px' width='auto'>"
+									data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0" a
+									class="link-primary"
+									href=""
+									target="_blank" rel="noopener noreferrer">
+									TM1637, 7 Segment Display
+								</a>
+
+
+							</td>
 							<td>1</td>
 						</tr>
 						<tr>
-							<td>Pushbutton Switch</td>
+							<td>
+								<a class="pj-part"
+								data-bs-content="<img src='assets/images/parts/ops_part_tactileSwitch.png' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0" a
+								class="link-primary"
+								href="https://drive.google.com/file/d/123mnk9r7DJ4KZHwoDXeP770cfP8593oN/view?usp=sharing"
+								target="_blank" rel="noopener noreferrer">
+									Switch, Tactile
+								</a>
+							</td>
 							<td>2</td>
 						</tr>
 						<tr>
-							<td>Buzzer, Piezo, 1.5V</td>
+							<td>
+								<a class="pj-part"
+								data-bs-content="<img src='assets/images/parts/ops_part_buzzer.webp' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0" a
+								class="link-primary"
+								href=""
```

Comment:
see comment on the TM code

Reply:
Resolved. Not intended; excess attributes deleted.